sandbox/docker/master/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim AS dependencies
+
+WORKDIR /tmp
+COPY ./requirements.txt /tmp/
+
+RUN apt-get update \
+        && apt-get install -y --no-install-recommends git \
+        && pip install -r requirements.txt \
+
+FROM python:3.9-slim
+
+WORKDIR /tmp
+
+COPY --from=dependencies /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
```

Comment:
I think we need to copy other files as well if we take a multi-stage build approach.

```
mypy-playground-app-1  | [E 211107 14:50:37 docker:73] docker api error
mypy-playground-app-1  |     Traceback (most recent call last):
mypy-playground-app-1  |       File "/app/mypy_playground/sandbox/docker.py", line 67, in run_typecheck
mypy-playground-app-1  |         await c.start()
mypy-playground-app-1  |       File "/usr/local/lib/python3.10/site-packages/aiodocker/containers.py", line 190, in start
mypy-playground-app-1  |         async with self.docker._query(
mypy-playground-app-1  |       File "/usr/local/lib/python3.10/site-packages/aiodocker/utils.py", line 309, in __aenter__
mypy-playground-app-1  |         resp = await self._coro
mypy-playground-app-1  |       File "/usr/local/lib/python3.10/site-packages/aiodocker/docker.py", line 275, in _do_query
mypy-playground-app-1  |         raise DockerError(response.status, json.loads(what.decode("utf8")))
mypy-playground-app-1  |     aiodocker.exceptions.DockerError: DockerError(400, 'OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "mypy": executable file not found in $PATH: unknown')
```

Reply:
```
> docker diff 69c954d3fb40  | grep -v "/usr/local/lib/python3.9|/root/.cache/pip|/usr/local/bin"
C /usr
C /usr/local
C /usr/local/lib
C /root
A /root/.cache
```